cloudbuild/cloudbuild.yaml

```diff
@@ -1,4 +1,13 @@
 steps:
+  - name: 'gcr.io/cloud-builders/gsutil'
+    id: 'get-build-num'
+    entrypoint: /bin/bash
+    args:
+      - -c
+      - |
+        chmod +x cloudbuild/periodic/get-and-update-build-num.sh &&
+        cloudbuild/periodic/get-and-update-build-num.sh
```

Comment:
Can we embed this script's logic in the `cloudbuild/Dockerfile` itself instead of heaving a separate step for it?

Reply:
If I put it in the Dockerfile, I have to authenticate or activate a service account. Doing it from the top-level GCB step allows me to use gsutil without problem.

Reply:
I think that because Docker file executed it has access for the same credentials as this script. May you try to do this in Docker file?